pkg/servicemodel/rc/v1/utils.go

```diff
@@ -226,3 +235,172 @@ func createRANParametersReportStyle2List() ([]*e2smrcies.ReportRanparameterItem,
+	headerFormat1 := format1.NewIndicationHeader(format1.WithEventConditionID(e2NodeInfoChangeID))
+	indicationHeaderAsn1Bytes, err := headerFormat1.ToAsn1Bytes()
+	if err != nil {
+		return nil, err
+	}
+	plmnID := c.getPlmnID()
+
+	cellInfoList := make([]*e2smrcies.E2SmRcIndicationMessageFormat3Item, 0)
+	for _, ncgi := range cells {
+		cell, err := c.ServiceModel.CellStore.Get(ctx, ncgi)
+		if err != nil {
+			return nil, err
+		}
+
+		nci := ransimtypes.GetNCI(ncgi)
+		nrCGI, err := pdubuilder.CreateNrCgi(plmnID.Value().ToBytes(), &asn1.BitString{
+			Value: utils.Uint64ToBitString(uint64(nci), 36),
+			Len:   36,
+		})
+		if err != nil {
+			return nil, err
+		}
+
+		cgi, err := pdubuilder.CreateCgiNRCgi(nrCGI)
+		if err != nil {
+			return nil, err
+		}
+
+		pci, err := c.getCellPCI(ctx, ncgi)
+		if err != nil {
+			return nil, err
+		}
+		cellPCI, err := pdubuilder.CreateServingCellPciNR(pci)
+		if err != nil {
+			return nil, err
+		}
+
+		earfcn, err := c.getARFCN(ctx, ncgi)
+		if err != nil {
+			return nil, err
+		}
+		cellArfcn, err := pdubuilder.CreateServingCellArfcnNR(earfcn)
+		if err != nil {
+			return nil, err
+		}
+
+		neighborCellList := make([]*e2smrcies.NeighborCellItem, 0)
+		for _, neighborNCGI := range cell.Neighbors {
+			neighborNci := ransimtypes.GetNCI(neighborNCGI)
+			neighborNrCGI, err := pdubuilder.CreateNrCgi(plmnID.Value().ToBytes(), &asn1.BitString{
+				Value: utils.Uint64ToBitString(uint64(neighborNci), 36),
+				Len:   36,
+			})
+			if err != nil {
+				return nil, err
+			}
+
+			neighborPci, err := c.getCellPCI(ctx, neighborNCGI)
+			if err != nil {
+				return nil, err
+			}
+
+			neighborEarfcn, err := c.getARFCN(ctx, neighborNCGI)
+			if err != nil {
+				return nil, err
+			}
+
+			neighborNrArfcn, err := pdubuilder.CreateNrArfcn(neighborEarfcn)
+			if err != nil {
+				return nil, err
+			}
+
+			nrFrequencyBandList := &e2smcommonies.NrfrequencyBandList{
+				Value: make([]*e2smcommonies.NrfrequencyBandItem, 0),
+			}
+
+			supportedSulbandList := make([]*e2smcommonies.SupportedSulfreqBandItem, 0)
+			supportedSulbandItem, err := pdubuilder.CreateSupportedSulfreqBandItem(1024)
+			if err != nil {
+				return nil, err
+			}
+			supportedSulbandList = append(supportedSulbandList, supportedSulbandItem)
+
+			frequencyBandItem, err := pdubuilder.CreateNrfrequencyBandItem(1, &e2smcommonies.SupportedSulbandList{
+				Value: supportedSulbandList,
+			})
+			if err != nil {
+				return nil, err
+			}
+
+			nrFrequencyBandList.Value = append(nrFrequencyBandList.Value, frequencyBandItem)
+
+			nrFrequencyInfo, err := pdubuilder.CreateNrfrequencyInfo(neighborNrArfcn, nrFrequencyBandList)
+			if err != nil {
+				return nil, err
+			}
+			nrFrequencyInfo.SetFrequencyShift7P5Khz(pdubuilder.CreateNrfrequencyShift7P5KhzTrue())
```

Comment:
As per the spec RC v01.01.05 and TS 38.473 v150800, I could not find FrequencyShift7P5Khz field. @eroshiva or @adibrastegarnia can you tell me which version you are referring to?